### PR TITLE
Update to repo-config 0.4.0

### PR DESCRIPTION
**Pull request by Marenz**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+Copyright © 2023 Frequenz Energy-as-a-Service GmbH
```
Comment (llucax) [inline LICENSE]: This shouldn't be updated.

File: mkdocs.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 site_name: "Frequenz channels"
 site_description: "Channel implementations for Python"
 site_author: "Frequenz Energy-as-a-Service GmbH"
-copyright: "Copyright © 2022 Frequenz Energy-as-a-Service GmbH"
+copyright: "Copyright © 2023 Frequenz Energy-as-a-Service GmbH"
```
Comment (llucax) [inline mkdocs.yml]: Nor this.

File: pyproject.toml [hunk 3]
```diff
@@ -1,11 +1,11 @@
 # License: MIT
-# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
```
Comment (llucax) [inline pyproject.toml]: Or this :D

**Review by llucax — COMMENTED** [review]
Except for the copyright year updates, LGTM.

**Comment by llucax** [comment]
Please also remember to set the milestone to the PR (I already did it).

**Comment by Marenz** [comment]
updated

**Review by llucax — CHANGES_REQUESTED** [review]
The year change is still there everywhere, maybe you forgot to push?

**Comment by Marenz** [comment]
hu? You can see my force-push removed it here? https://github.com/frequenz-floss/frequenz-channels-python/compare/1787c949724100c0fd9f652b3f70e287b6c45448..eead2f927b665598242cb57a0520f00ac338d29f

**Comment by Marenz** [comment]
And I don't see it in the files-changed tab either...

**Review by llucax — APPROVED** [review]
Now it looks fine. Maybe it was a cache issue on my side?

**Pull request merged**